CHANGELOG.md

```diff
@@ -1,4 +1,18 @@
-## 3.0.3 (Unreleased)
+## 3.0.4 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-7.0.1724**
+- Supported Terraform version: **v1.x**
+
+### Features:
+#### Settings
+1. Implemented a new resource to support Controller Security Group Management:
+   - **aviatrix_controller_access_allow_list_config**
+
+### Enhancements:
+1. Added support of space character ``descripton`` in **aviatrix_firewall_policy**
```

Comment:
Added support for whitespace characters in **aviatrix_firewall_policy** ``description``